packages/hoprd/releases-schema.json

```diff
@@ -1,40 +1,42 @@
 {
-  "type": "array",
+  "type": "object",
   "items": {
     "type": "object",
-    "properties": {
-      "id": {
+    "patternProperties": {
```

Comment:
How are the schema and releases.json changes relevant to fixing test timeout?

Reply:
They are relevant since the schema test failed, as reported in #2858